mantle/platform/cluster.go

```diff
@@ -125,18 +125,20 @@ func (bc *BaseCluster) appendSSH(m Machine) error {
 	defer sshConfig.Close()
 	sshBuf := bufio.NewWriter(sshConfig)
 
-	machIP := m.IP()
-	idx := strings.LastIndex(machIP, ":")
-	if idx == -1 {
-		panic(fmt.Sprintf("Malformed machine IP %s", machIP))
+	_, err = fmt.Fprintf(sshBuf, "Host %s\n", m.ID())
+	if err != nil {
+		return err
 	}
-	_, err = fmt.Fprintf(sshBuf, `Host %s
-  HostName %s
-  Port %s
+	host, port, err := net.SplitHostPort(m.IP())
```

Comment:
`err` isn't being used for anything, might as well make it `_` and not waste the space.

Reply:
Argh, I try to obsessively check for errors but Golang makes it sooo easy to mess up.